src/routing/http-routing-v1.md

```diff
@@ -131,6 +181,53 @@ The client SHOULD be able to make a request with `Accept: application/x-ndjson`
 
 Each object in the `Peers` list is a record conforming to the [Peer Schema](#peer-schema).
 
+### `PUT /routing/v1/peers`
+
+#### `PUT` Request Body
+
+```json
+{
+  "Providers": [
+    {
+      "Schema": "announcement",
+      ...
+    }
+  ]
+}
+```
```

Comment:
Consistent with the GET?

```suggestion
```json
{
  "Peers": [
    {
      "Schema": "announcement",
      ...
    }
  ]
}
```
```